.storybook/shared/decorators/with-background.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const withBackground = (Story, context) => {
+    const colorMode = context.globals.theme;
+    const isDark = colorMode === 'dark';
+
+    if (isDark) {
+        document.documentElement.style.background = '#0E0E0E';
+        document.body.style.background = '#0E0E0E';
+    } else {
+        document.documentElement.style.background = '#ffffff';
```

Comment:
Same as before

Reply:
sure thing, updated the code, thanks :pray: